sdk-api-src/content/d3d12sdklayers/nn-d3d12sdklayers-id3d12debug.md

```diff
@@ -1,7 +1,8 @@
 ---
 UID: NN:d3d12sdklayers.ID3D12Debug
 title: ID3D12Debug (d3d12sdklayers.h)
-description: A debug interface controls debug settings and validates pipeline state. It can only be used if the debug layer is turned on.
helpviewer_keywords: ["ID3D12Debug","ID3D12Debug interface","ID3D12Debug interface","described","d3d12sdklayers/ID3D12Debug","direct3d12.id3d12debug"]

```

Comment:
Should I restore the ^M here? I'm not sure why it went away

Reply:
I've seen that happen a lot; I think we should leave it.